charts/tuf/values.yaml

```diff
@@ -0,0 +1,41 @@
+forceNamespace: ""
+imagePullSecrets: []
+serviceAccountName: tuf-sa
```

Comment:
Can this be changed to `tuf` as its already implied that it is an SA?

Reply:
Yes. Addressed in 22ad794